extensions/wikia/UserLogin/js/UserSignup.js

```diff
@@ -43,6 +44,29 @@
 				.add(inputs.birthmonth)
 				.add(inputs.birthyear)
 				.on('change.UserSignup', this.validator.validateBirthdate.bind(this.validator));
+		},
+
+		/**
+		 * We'd like to understand better how many pageviews are generated by bots on UserSignup pages.
+		 * This simple code attaches to few events which we believe are not usually fired by bots,
+		 * then if event is detected we track an event in our analytics systems.
+		 */
+		setupHumanTracking: function () {
+			$(document).on(
```

Comment:
Is there any way to flatten the callback chain here? :) Defining callbacks at the top of this method might be a good start :)